examples/Advanced_Sampling_Introduction.ipynb

```diff
@@ -0,0 +1,1853 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def plot_one_free_energy(result):\n",
+    "  histogram_log = result.callbacks[0]\n",
+    "  \n",
+    "  hist, edges = histogram_log.get_histograms(bins=30, range=[(0,4)])\n",
+    "  fig, ax = plt.subplots()\n",
+    "  ax.set_xlabel(r\"$\\xi$ $[\\sigma]$\")\n",
+    "  ax.set_ylabel(r\"A(\\xi)\")\n",
+    "  ax.set_xlim((0, 4))\n",
+    "\n",
+    "  x = edges[0][:-1] + np.diff(edges[0])/2\n",
+    "\n",
+    "  weight = np.exp(-kspring/2*(x-2)**2)\n",
+    "  unbiased_distribution = hist/weight\n",
+    "  unbiased_distribution /= integrate.simpson(unbiased_distribution, x)\n",
+    "\n",
+    "  estimated_profile = -kBT * np.log(unbiased_distribution)\n",
+    "  constant_C = -np.min(estimated_profile) + np.min(potential(x)[0])\n",
+    "  ax.plot(x, estimated_profile + constant_C, label=r\"estimated $A(\\xi)$\")\n",
+    "  \n",
+    "  ax.plot(x, potential(x)[0], label=r\"true $A(\\xi)$\")\n",
+    "\n",
+    "  ax.legend(loc=\"best\")\n",
+    "  fig.show()\n",
+    "plot_one_free_energy(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "e2YtfQlQ8jO9"
+   },
+   "source": [
+    "That estimation is not bad.\n",
+    "We get the approximate right shape in the middle and that could be further improved by running the sampling trajectory longer.\n",
+    "[Try it out!]\n",
+    "\n",
+    "But there are still some issues because we still cannot sample the entire space:\n",
+    "\n",
+    "*  we do not find the minima at 0\n",
+    "*  the up trend on the right is uncovered\n",
+    "*  the second minima is overestimated\n",
+    "\n",
+    "Can we bias simulations in these regions too, to improve sampling coverage in these regions too?\n",
+    "\n",
+    "## Umbrella Sampling\n",
+    "\n",
+    "We want to find the free-energy profile along a given path in the space for collective variables. Usually, this path can be multidimensional.\n",
+    "\n",
+    "Example dihedral angles of Alanine Dipeptide. [PySAGES Alanine Dipentide examples](https://colab.research.google.c
```

Comment:
```suggestion
   "collapsed_sections": [
     "T-Qkg9C9n7Cc"
   ],
```
Would it make sense to hide the preamble by default?

Reply:
yup, no noise needed.

Reply:
We could add a note saying that people interested in seeing how to set up the environment can expand the section.

Reply:
true, but I don't think it is necessay.
Let me go through it.
I will push it collapsed.